### PR TITLE
conda release steps

### PR DESCRIPTION
**Pull request by md-arif-shaikh**
Added steps to add recipe to conda-forge

### CODE REVIEW

File: README_developers.md [hunk 1]
```diff
@@ -29,3 +29,17 @@ Note: Currently, only Vijay and Arif can do this, as they are the only ones with
 python -m build
 twine upload dist/*
 ```
+## Create Conda recipe
+**NOTE:** This is needed to be done only once to add the recipe to conda-forge/staged-recipe.
+To update the conda release follow conda release section below.
```
Comment (vijayvarma392) [inline README_developers.md:34]: Where is this section? Not sure I see it...
Isn't the idea that this does not need to be done for each new pypi release? This sentence seems to say the opposite.

**Review by vijayvarma392 — CHANGES_REQUESTED** [review]

**Comment by vijayvarma392** [comment]
Please remember to add conda instructions here: https://github.com/vijayvarma392/gw_eccentricity#installation
And please test that it works on your laptop.

File: README_developers.md (hunk 1 above)
Comment (md-arif-shaikh) [reply README_developers.md:34]: added the instructions to upgrade to new version

**Comment by vijayvarma392** [comment]
Please explain in the readme when this needs to be done:
image

Manually for every PyPi release? Or is it done automatically?

**Comment by md-arif-shaikh** [comment]
I don't think it will be automatically creating a new conda-release whenever we upgrade the PyPI version. If you look at the recipe, i.e., the `meta.yaml` file linked above, we are pinning the recipe to a particular version and therefore to upgrade the conda version we need to update the version in `meta.yaml`. There is probably a way to automate this but the instructions  written above are from the feedstock repo, the link is also in the instructions.

**Comment by vijayvarma392** [comment]
Ok, just say explicitly that this needs to be done after every PyPI release, just for the future people.

**Comment by md-arif-shaikh** [comment]
> Ok, just say explicitly that this needs to be done after every PyPI release, just for the future people.

updated.

**Comment by md-arif-shaikh** [comment]
ready to be merged? @vijayvarma392

**Review by vijayvarma392 — APPROVED** [review]

**Pull request merged**